src/puppetlabs/puppetdb/nio.clj

```diff
@@ -0,0 +1,14 @@
+(ns puppetlabs.puppetdb.nio
+  (:import
+   [java.nio.file Path Files Paths StandardCopyOption]))
+
+(def copt-atomic StandardCopyOption/ATOMIC_MOVE)
```

Comment:
Right, stolen from stockpile, but I'm fine with something better; I'd just tried to pick something simple/short and opted (at least atm) for direct bindings so we don't have a hash table look up for a simple #define -- suppose that could be argued overoptimization.